www/src/components/landingPage/banner.astro

```diff
@@ -73,7 +73,7 @@ import ClipboardSelect from "./ClipboardSelect";
                   <div
                     class="relative flex items-center rounded-lg border border-t3-purple-200/20 bg-t3-purple-100/10 px-2 py-2 text-sm md:px-3 md:py-3 md:text-lg lg:px-5 lg:py-4 lg:text-xl"
                   >
-                    <code class="mr-12">npx create t3-app@latest</code>
+                    <code class="mr-12">npx create-t3-app@latest</code>
```

Comment:
This one is indeed incorrect - seems to have been missed when #1612 was reviewed. but personally I think it would be better to stay consistent and use `npm create` here, as it's what we're doing everywhere else.

```suggestion
                    <code class="mr-12">npm create t3-app@latest</code>
```